### PR TITLE
Makefile generation fixes for .stringmaptable

### PR DESCRIPTION
**Pull request by maxim-zhao**
* Emit stringmaptable filenames to makefile
* Do not fail the build when the file is not found, if in makefile mode

Relates to #446

### CODE REVIEW

**Comment by maxim-zhao** [comment]
The build failure above is erroneous (Azure warning about some infra changes), but it seems I can't trigger a rebuild.

**Comment by maxim-zhao** [comment]
Fixed merge conflicts.

File: include_file.c [hunk 1]
```diff
@@ -509,38 +510,37 @@ int print_file_names(void) {
 
   struct incbin_file_data *ifd;
   struct file_name_info *fni;
-
+  struct stringmaptable *smt;
+  int is_first_line = YES;
```
Comment (maxim-zhao) [inline include_file.c:514]: Note that I changed the way this method works to simplify the logic about when to add space-newline-tab.

**Comment by vhelin** [comment]
Thanks!

**Pull request merged**